os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -287,7 +287,7 @@ static void check_sp_corruption(uint32_t sp, uint32_t *stackbase, uint32_t *stac
 {
 	lldbg_noarg("===========================================================\n");
 	lldbg_noarg("Asserted task's stack details\n");
-	lldbg_noarg("===========================================================\n");	
+	lldbg_noarg("===========================================================\n");
```

Comment:
Please revert this extra space at the end

Reply:
There is no extra space at the end rather I have removed it.